examples/all-clusters-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -55,7 +56,8 @@ using namespace ::chip::System;
 using namespace ::chip::DeviceLayer;
 using namespace chip::app;
 
-constexpr uint32_t kIdentifyTimerDelayMS = 250;
+constexpr uint32_t kIdentifyTimerDelayMS     = 250;
+constexpr uint32_t kInitOTARequestorDelaySec = 3;
```

Comment:
Is 3 seconds sufficient amount of time for WiFi to start?

Reply:
Wifi is already started, also got ip. After 3s we are initialising OTA.